test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,35 @@
+package util.fixtures;
+
+import entities.Client;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class ClientServiceFixtures {
+  public static final Long FAKEID = Long.valueOf(10L);
+
+  public static List<Client> buildClients(Integer size) {
+    List<Client> clients =
+        IntStream.range(0, size)
+            .mapToObj(i -> ClientServiceFixtures.buildClient(Long.valueOf(i)))
+            .collect(Collectors.toList());
+    return clients;
+  }
+
+  public static Client buildClient() {
+    return buildClient((Client) null);
+  }
+
+  public static Client buildClient(Long id) {
+    var clientExample = new Client();
+    clientExample.setId(id);
+    return buildClient(clientExample);
+  }
+
+  public static Client buildClient(Client clientExample) {
+    var client = new Client();
+    UserFixtures.buildUser(client, clientExample);
+    BaseEntityFixtures.buildBaseEntity(client, clientExample);
```

Comment:
Why do you call this here? Shouldn't you call this within user fixture class instead?

Reply:
To separate the fixtures methods, but solved